### DISCUSSION TITLE
OpenVDB + Apple M1

### DISCUSSION THREAD

**Discussion by danrbailey** [AcademySoftwareFoundation/openvdb#1144] [General]
I just tried to build OpenVDB on Apple M1. I built the vdb_print binary and the unit tests successfully and all pass. OpenVDB itself needed no modifications to run on M1.

I tested with these dependencies:

* TBB 2020.2
* Boost 1.73.0
* Blosc 1.5.0 / 1.21.0 *

\* Blosc 1.5.0 requires a few modifications to build it (bumping minimum CMake to 3.0 and a few minor fixes related to SSE). Blosc 1.21.0 builds without modifications but the TestStreamCompression.testPagedStreams unit test fails due to some likely benign offset checks.

Just a quick test for now - I didn't assess performance or try with Half support, the other binaries, the Python module or the Houdini plugin.

**Comment by paulkre** [comment]
How did you manage to fix the issues regarding SSE while building Blosc?

**Reply by danrbailey** [reply]
If you can, I would suggest using Blosc 1.21.0 as it doesn't require any fixes to SSE. We're soon going to be updating our recommendations to use this newer version anyway.
